onnx/common/ir_pb_converter.cc

```diff
@@ -204,6 +204,13 @@ std::vector<Dimension> tensorShapeProtoToDimensions(const ONNX_NAMESPACE::Tensor
   return dims;
 }
 
+void createDummyValue(std::unique_ptr<Graph>& g, const std::string& name, std::unordered_map<std::string, Value*>& value_by_name_of) {
+  auto* undef = g->create(kCaptured, 1);
```

Comment:
What exactly does `kCaptured` mean? Maybe it is relevant to the other two original calls, but not sure it means the same for undefined valueinfos.

Reply:
Good question. It seems to me that `kCaptured` is used for an inner (nested) graph that some inputs/outputs might not be visible in value_by_name_of so it creates a dummy node for them with `kCaptured` type.